route-api/main.ts

```diff
@@ -28,3 +31,114 @@ mqtt_client.on("message", (_, message) => {
 mqtt_client.on("error", (error) => {
   console.log(error);
 });
+
+const app = new Application();
+
+// Request an Here-Transit-API
+async function getRoute(
+  origin,
+  destination,
+  arrivalTime,
+  departureTime,
+  lang,
+  units,
+  changes,
+  alternatives,
+  modes,
+  pedestrianSpeed,
+  apiKey,
+) {
```

Comment:
Use typed object as parameter with types:

```suggestion
async function getRoute(options: GetRouteOptions, apiKey: string) {
```

with typedefinition:

```ts
type GetRouteOptions {
  origin: string;
  destination: string;
  arrivalTime?: string;
  // ...
}
```